src/components/Filter/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './Filter.container'
```

Comment:
Attention, tu doit surement avoir des warning lorsque tu lance ton application. Vérifie ce que te dis ton compilateur.

Ici par exemple, il te manque un retour ligne. Tous les fichiers sources doivent toujours terminer par un retour ligne final (convention de git qui permet d'éviter les différence de commit inutile).

Reply:
```suggestion
export { default } from './Filter.container'
```
J'ai aucun warning, voila les captures d'écran des 2 serveurs et console de Chrome Tools:

<a href="https://ibb.co/kKpWDL"><img src="https://preview.ibb.co/njh7nf/2018-11-05-20-01-34.png" alt="2018-11-05-20-01-34" border="0"></a>
<a href="https://ibb.co/gT7wf0"><img src="https://preview.ibb.co/bCkSnf/2018-11-05-20-01-52.png" alt="2018-11-05-20-01-52" border="0"></a>
<a href="https://ibb.co/cF2jYL"><img src="https://preview.ibb.co/mF9900/2018-11-05-20-02-09.png" alt="2018-11-05-20-02-09" border="0"></a><br />

Je vais faire nouveau PR avec les retours à la ligne des fichiers

Reply:
Ha d'accord, étrange je pensais que ce genre de chose était check. Autant pour moi :-)